hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/receiver/kafka/KafkaSingleThreadedMessageReceiver.java

```diff
@@ -93,7 +94,7 @@ public KafkaSingleThreadedMessageReceiver(KafkaConsumer<byte[], byte[]> consumer
     public Optional<Message> next() {
         try {
             if (readQueue.isEmpty()) {
-                ConsumerRecords<byte[], byte[]> records = consumer.poll(pollTimeout);
+                ConsumerRecords<byte[], byte[]> records = consumer.poll(Duration.ofMillis(pollTimeout));
```

Comment:
What about s/`pollTimeout`/`pollTimeoutMillis` as in `KafkaProperties.java` ?

Reply:
I know it's a bit confusing at first but ```KafkaProperties.java``` are used in context of Spring app. In this case we have ```hermes-management``` package which uses Spring and another set of consumers for example for enabling moving offset for retransmission. ```KafkaSingleThreadedMessageReceiver.java``` on the other hand uses another set configuration not connected to ```hermes-management``` ;)

Reply:
We can talk about it for a while, when you will be back :)